api/v1alpha1/stage_types.go

```diff
@@ -300,6 +300,7 @@ type FreightSources struct {
 	// AvailabilityStrategy specifies the semantics for how requested Freight is
 	// made available to the Stage. This field is optional. When left unspecified,
 	// the field is implicitly treated as if its value were "OneOf".
+	// Accepted Values: "All", "OneOf", ""
```

Comment:
Is this the only place where we use enumerated values without defining what they are?

It is probably worthwhile to explain, as we do for image selection strategies, what the meaning of each value is.

Reply:
> I don't see explanation in the imageSelectionStrategy:

🤦‍♂️ It's defined somewhere internally:

https://github.com/akuity/kargo/blob/dfc75c46d0467ba1e98fec7abf9907976bfd0185/internal/image/selector.go#L13-L39

These explanations really ought to be copied into `warehouse_types.go` so that they'll show up in the generated docs.

Looks like similar is true for commit selection strategies are also missing explanations of their values.